tests/framework/e2e/etcd_spawn_cov.go

```diff
@@ -35,40 +32,21 @@ var (
 	coverDir = testutils.MustAbsPath(os.Getenv("COVERDIR"))
 )
 
-func SpawnCmdWithLogger(lg *zap.Logger, args []string, envVars map[string]string, name string) (*expect.ExpectProcess, error) {
-	cmd := args[0]
-	env := mergeEnvVariables(envVars)
-	switch {
-	case strings.HasSuffix(cmd, "/etcd"):
-		cmd = cmd + "_test"
```

Comment:
Do you understand why the ` _test `suffix was added in the first place? 

cc @ptabor

Reply:
Yes, the coverage scenario compiles separate binaries with `_test` suffix.

Reply:
I was thinking removing the suffix "_test" may cause some issue. I just realized that you do not remove the suffix, which is hard coded in `initBinPathCov`.